cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_diffRegionDiffNames.ump

```diff
@@ -0,0 +1,27 @@
+class X {
```

Comment:
In my test --> assertNoWarnings("487_parallelStateMachines_diffRegionDiffNames.ump");